src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -69,15 +69,7 @@ public static class Helpers
 
         public static void WaitForSocketShutdown(Socket socket)
         {
-            if (PlatformDetection.IsWindows || PlatformDetection.IsOSX)
```

Comment:
So I found that the tests intermittently failed on Windows if I didn't have this code. See https://github.com/dotnet/corefx/issues/18188

Is this something you found?